### PR TITLE
Support for comparisons between SympyConstants

### PR DESCRIPTION
**Pull request by mmatera**
Add support to compare SympyConstants

### CODE REVIEW

File: mathics/builtin/__init__.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
     algebra, arithmetic, assignment, attributes, calculus, combinatorial, compilation,
     comparison, control, datentime, diffeqns, evaluation, exptrig, functional,
     graphics, graphics3d,
-    image, inout, integer, iohooks, linalg, lists, logic,
+    # image,
+    inout, integer, iohooks, linalg, lists, logic,
```
Comment (weakit) [inline mathics/builtin/__init__.py:10]: This right here is what I'm talking about.

**Review by weakit — COMMENTED** [review]
I may be wrong, but there seem to be a few unrelated changes in a few files. And a removed picture in the documentation.
Could you go over them?

It'd also be helpful if you could fix the tests and add a few new ones. I'll take a look at this tommo and maybe try to help.

File: mathics/builtin/__init__.py (hunk 1 above)
Comment (mmatera) [reply mathics/builtin/__init__.py]: Yes, my mistake. I always comment out this module for local testings...

Comment (weakit) [reply mathics/builtin/__init__.py]: No probs, nobody's perfect!

**Comment by weakit** [comment]
This wouldn't really work for something like `Max[Pi, Pi + 1]`, would it?

**Comment by mmatera** [comment]
> This wouldn't really work for something like `Max[Pi, Pi + 1]`, would it?

No as it is. This PR requires more work, but I didn't find the time to do it. The idea was first to identify where in the code the comparison is done, and then, look for the smartest way to extend it using sympy.

**Comment by weakit** [comment]
> This PR requires more work, but I didn't find the time to do it.
I think I'll try working on it later today.

**Comment by rocky** [comment]
@mmatera is this still relevant?

**Comment by weakit** [comment]
I don't think so. PR #1000 closed #965.

**Pull request closed without merging**